src/components/pages/Profile/History/index.tsx

```diff
@@ -10,7 +10,7 @@ import styles from './index.module.css'
 import OceanProvider from '../../../../providers/Ocean'
 import { useWeb3 } from '../../../../providers/Web3'
 import { useAddressConfig } from '../../../../hooks/useAddressConfig'
-import { useSiteMetadata } from '../../../../hooks/useSiteMetadata'
+import { allowDynamicPricing } from '../../../../../app.config'
```

Comment:
I see why this looks a little cleaner, but I actually like to use hooks when it comes to the app.config. It has the benefit of being able to use typings defined for useSiteMetadata().appConfig for example.

There is one exception which is if you have to use a config var outside a react component. The probably safest way would be to have the variable be passed to the function in that case, but depending on the complexity added & readability lost I would argue that in some cases a direct import from app.config makes sense.

Reply:
> Code looks fine! We should replace the last (satellites) animation with an image of the foreverontheblockchain web app though. It looks less impressive but it gives a better impression of the actual product. We can do this in a separate PR.

@oceanByte We could maybe utilize the animation of the logo that you have on page load in the foreverontheblockchain home page to make it a little bit more dynamic